apps/condo/domains/acquiring/schema/SumPaymentsService.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Generated by `createservice acquiring.SumPaymentsService`
+ */
+
+const { GQLCustomSchema } = require('@condo/keystone/schema')
+const access = require('@condo/domains/acquiring/access/SumPaymentsService')
+const { GqlWithKnexLoadList } = require('../../common/utils/serverSchema')
+const Big = require('big.js')
+
+
+
+const SumPaymentsService = new GQLCustomSchema('SumPaymentsService', {
+    types: [
+        {
+            access: true,
+            type: 'type SumPaymentsOutput { sum: String! }',
+        },
+    ],
+
+    queries: [
+        {
+            access: access.canSumPayments,
+            schema: 'sumPayments(where: PaymentWhereInput!): SumPaymentsOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { where } = args
+                const paymentLoader = new GqlWithKnexLoadList({
+                    listKey: 'Payment',
+                    fields: 'id',
+                    where: where,
+                })
+                const idObjects = await paymentLoader.load()
+                const aggregate = await paymentLoader.loadAggregate('SUM(amount) as "amountSum"', idObjects.map(({ id }) => id))
```

Comment:
@dkoviazin can we use it without ids array?

Reply:
then we will need to convert graphql queries to sql
we don't need it until we migrate to prizm